lib/active_stash/assess.rb

```diff
@@ -11,20 +11,23 @@ class Assess
     REPORT_FILE_NAME = "active_stash_assessment.yml"
     DOCS_BASE_URL = "https://docs.cipherstash.com/assess/checks"
 
-    def initialize(quiet: false)
-      if !defined?(Rails)
-        raise RailsUndefinedError, "ActiveStash Assess can currently only be used in Rails projects"
-      end
-
-      Rails.application.eager_load!
-
-      @assessment_path = self.class.assessment_path
+    def initialize(quiet: false, models: default_models, report_dir: Rails.root)
```

Comment:
Moving `models` and `report_dir` into constructor args to make testing easier. This lets us test Assess without having to pull in a full Rails app (or multiple Rails apps).

The tests rely on Ruby lazily evaluating the defaults here.